public/app/screens/main/navigation/navigation.jsx

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var React = require('react'),
+  Link = require('react-router').Link,
+  StyleMixin = require('mixins/style-mixin');
+
+var Navigation = React.createClass({
+  mixins: [StyleMixin(require("./style.scss"))],
+  render: function () {
+    // TODO: Use mixins instead of CSS classes
+    // TODO: Pass number of elements to mixin instead of specifying class name
+    var classes = "cc_navigation icon-bar two-up " + this.props.orientation;
+    
+    var items = this.props.items.map(function (item, i) {
+      return (
+        <Link to={item.linkTo} key={i} className="cc_navigation__link item">
```

Comment:
Yes. The answer is the same as to the same question you did recently. It's needed by react: https://coderwall.com/p/jdybeq/the-importance-of-component-keys-in-react-js